cfp.md

```diff
@@ -34,18 +34,62 @@ Contributions may overlap with any of the following areas:
+The review process is **blind** (not double-blind);
+please do not submit anonymized manuscripts.
+
+### Submission Site
+Please submit your paper at the [Manuscript Central Submission Site](https://mc.manuscriptcentral.com/les-ieee).
+Click on "**Author**," create a new manuscript, and select "**Type**" as "**SI:TCRS24**".
+
+### Conflicts of Interest Requirements
+In addition to the manuscript, the authors are required to submit a **Cover Letter** indicating the **Conflicts of Interest (COIs)** with the program committee members through Manuscript Central.
+The authors can submit the cover letter during the Submission **Step 6: Details & Comments**, using a textbox under **Write Cover Letter**.
+Even if the authors do not have any COIs, they are still required to submit a cover letter confirming that they have no COIs.
+
+### Acceptance Tiers
+At least one of the authors for each paper must fully register and present the accepted paper at the workshop.
+TCRS '24 will have two tiers of final acceptance.
+
+1. *Present-only* acceptance.
+2. *Present-and-publish* acceptance.
+
+The *Present-only* tier acceptance will be given to the submissions that meet the TCRS workshop quality requirements.
+The *Present-and-publish* tier acceptance will be given to the submissions that meet the quality requirements for IEEE ELS journal publication as well as the quality requirements for the TCRS workshop.
 
-### Submission site
+All papers with any of these two tiers of acceptance will be presented at the workshop and distributed as proceedings of the workshop.
+However, only the papers with the *Present-and-publish* tier acceptance will be published in the IEEE ESL special issue of TCRS.
 
-TBA
+### Review Process
+Authors of the submissions will be notified with the first round of reviews.
+Below are the subsequent processes for accepted papers.
+
+1. Authors of the submissions with *Present-only* acceptance will be asked to submit the final PDFs that will be included in the "*proceedings to be distributed*."
+The review result from IEEE ESL will be a "reject" for the submission with *Present-only* acceptance; however, the authors will receive a separate notice for *Present-only* acceptance.
+
+2. Authors of the submissions with *Present-and-publish* acceptance will receive a "minor revision" decision from IEEE ESL and will be required to submit the final camera-ready PDF and a video graphical abstract which is a short video presenting the paper.
+Both the final PDF manuscript and the video graphical abstract will be published by IEEE ESL.
+See [here](https://ieee-ceda.org/publication/esl/esl-paper-submission) for more information on the video graphical abstract.
+
+Final submissions with *Present-and-publish* acceptance, including both the final camera-ready PDF and the video graphical abstract, will be reviewed in the second round.
```

Comment:
No I'm confused again. Camera-ready instructions are above, but now we're talking about a second round of reviews?

Reply:
Maybe we should not call it camera-ready, then. The second submission is indeed subject to a second round of reviews. Let me work on this a bit.